.cirrus.yml

```diff
@@ -207,6 +209,14 @@ integration_task:
             DISTRO_NV: "${FEDORA_NAME}"
             IMAGE_NAME: "${FEDORA_CACHE_IMAGE_NAME}"
             STORAGE_DRIVER: 'vfs'
+            BUILDAH_RUNTIME: crun
+            RUNTIME_N: " using crun"
+        - env:
+            DISTRO_NV: "${FEDORA_NAME}"
+            IMAGE_NAME: "${FEDORA_CACHE_IMAGE_NAME}"
+            STORAGE_DRIVER: 'vfs'
+            BUILDAH_RUNTIME: runc
+            RUNTIME_N: " using runc"
```

Comment:
We have dropped all runc testing not to long ago in podman. It was a huge maintenance headache.

Every image update now has yet another big failure point which causes regressions. As the person who currently manages all the updates I am strongly against making my life harder with these things.

Reply:
Almost every CI image update contains one or more regressions in our dependencies, kernel, systemd, golang, crun, glibc, well you get the point. By testing with runc every runc update is now a new potential failure point that then blocks image updates and requires me (or whoever does the image update) to do the work of investigating the test failure, reporting the bug and/or adding a work around or skip to test. Can that be done? Sure, but we decided over a year ago that we no longer want to test runc (https://github.com/containers/podman/pull/22706) and I at least am not interested in doing a ton of extra work for it.

Reply:
Then those can be marked as not required for the "Total Success" check.  But the idea of fixing this bug that runc exposes, and then not taking available steps to help ensure that we won't encounter something similar again, doesn't sit well with me.

Reply:
We could have caught this bug earlier.  If we need to have a conversation about that, then I think we should have the conversation.

Reply:
I understand @Luap99's concerns about this being yet another failure point that may or may not block releases, or that adds additional work. That being said, we use runc extensively with podman, so this is something that would be helpful in the long run. I can take up the work that @Luap99 is referring to with respect to runc and any issues arising out of it, if that seems to help, and take this forward.